en/guide/taking_off_landing.md

```diff
@@ -0,0 +1,220 @@
+# Take-off and Landing (and Other Actions)
+
+DroneCore provides the [Action](../api_reference/classdronecore_1_1_action.md) class for commanding the vehicle to arm, takeoff, land, return home and land, and disarm. 
+
+Most of the methods have both synchronous and asynchronous versions. The methods send commands to a vehicle, and return/complete with the vehicle's response. It is important to understand that a successful response indicates whether or not the vehicle intends to act on the command, not that it has finished the action (e.g. arming, landing, taking off etc.).
+
+> **Note** The implication is that you may need to monitor for completion of actions!
+
+<span></span>
+> **Note** All the code fragments below refer to the object `device`, which is a [connected device/vehicle](guide/connections.md).
+
+
+## Taking Off
+
+The recommended way to take off using DroneCore (and PX4) is to use either of the [takeoff()](../api_reference/classdronecore_1_1_action.md#classdronecore_1_1_action_1ae159c78b1a4056137187dc6e6d878539) or [takeoff_async()](../api_reference/classdronecore_1_1_action.md#classdronecore_1_1_action_1a2aec10a2b14f5e82f05edc6e2feac83e) methods. If a takeoff command is accepted the vehicle will change to the [Takeoff mode](https://docs.px4.io/en/flight_modes/takeoff.html), fly to the takeoff altitude, and then hover (in takeoff mode) until another instruction is received. 
+
+> **Note** PX4/DroneCore also provides other ways to take off:
+> - A copter or VTOL will take off automatically if a mission is started (fixed-wing will not). 
+> - You can also take off by manually driving the vehicle using the offboard API. 
+
+The vehicle will only take off once *armed*, and can only arm once it is "healthy" (has been calibrated, the home position has been set, and there is a high-enough quality GPS lock). After it starts flying, code needs to check that takeoff is complete before sending additional instructions.
+
+
+### Health Check
+
+The code fragment below shows very simple code to synchronously poll for health status (using [Telemetry:health_all_ok()](../api_reference/classdronecore_1_1_telemetry.md#classdronecore_1_1_telemetry_1a630c91d8067e4084c4f303513a0aeb29)) prior to arming.
+
+```cpp
+// Wait until health is OK and vehicle is ready to arm
+while (device.telemetry().health_all_ok() != true) {
+    std::cout << "Vehicle not ready to arm ..." << std::endl;
+    std::this_thread::sleep_for(std::chrono::seconds(1));
+}
+```
+
+The code fragment below performs the same task, but additionally exits the app if calibration is required reports what conditions are still required before the vehicle is "healthy":

```

Comment:
@julianoes  IMO This approach is much better than the default because it exits early in the case where vehicle cannot recover (requires calibration). Should we use this in examples?

Reply:
Yes it is. I'd welcome your proposal for a more concise approach, but this will do for now.